sdk/js/src/service/devices/index.js

```diff
@@ -17,7 +17,6 @@
 import traverse from 'traverse'
 import Marshaler from '../../util/marshaler'
 import Device from '../../entity/device'
-import randomByteString from '../../util/random-bytes'
```

Comment:
Should we remove the file as well then?

Reply:
<img width="682" alt="Screenshot 2019-08-28 at 14 18 12" src="https://user-images.githubusercontent.com/16374166/63852313-ab748000-c9a0-11e9-8945-bcfe43fb2c75.png">